### PR TITLE
add item_discrimination()

### PR DESCRIPTION
**Pull request by strengejacke**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -8,7 +8,6 @@ bibliography: paper.bib
 ```{r, echo = FALSE}
 knitr::opts_chunk$set(
   collapse = TRUE,
-  tidy.opts = list(width.cutoff = 100),
```
Comment (IndrajeetPatil) [inline README.Rmd:11]: For inconsequential commits like these, please use `[skip ci]` in your commit messages (e.g. `remove tidy opts [skip ci]`). We really don't need to run 24 GHA workflows to check a change like this.

**Comment by IndrajeetPatil** [comment]
**Squash and merge**. Please squash and merge. Let's have a clean git history. 😭😭😭

**Comment by strengejacke** [comment]
I set squash and merge as default in VS Code, but branch protection prevents me from merging:

image

**Comment by strengejacke** [comment]
Can't we change the default in the GH web interface, too?

**Comment by IndrajeetPatil** [comment]
But the GH web interface already defaults to _Squash and merge_. I had changed settings for all repos a while back for this. I am surprised that this is not what you see when you make a PR:

Screenshot 2023-01-18 at 19 16 31

**Comment by strengejacke** [comment]
Yes, I remember this being the default some time ago, but now the default is just "merge" again. Not sure why this changed?
Screenshot_20230118-194721

**Comment by rempsyc** [comment]
The other day when merging Etienne's PR, I almost made the same mistake. I thought the default was squash and merge so almost didn't look but I see the same thing as Daniel from the web interface.

**Pull request merged**